otelcollector/deploy/addon-chart/ccp-metrics-plugin/templates/ama-metrics-deployment.yaml

```diff
@@ -0,0 +1,235 @@
+            - /configmap-watcher
+          args:
+            - --configmap-namespace=kube-system
+            - --configmap-name=ama-metrics-settings-configmap
+            - --kubeconfig-file=/etc/kubernetes/kubeconfig/kubeconfig.yaml
+            - --settings-volume=/etc/config/settings
+          resources:
+            limits:
+              cpu: 20m
+              memory: 40Mi
+            requests:
+              cpu: 10m
+              memory: 20Mi
+          volumeMounts:
+            - name: kubeconfig
+              readOnly: true
+              mountPath: /etc/kubernetes/kubeconfig
+            - name: settings-vol-config
+              mountPath: /etc/config/settings
+        - name: msi-adapter
+          image: "mcr.microsoft.com/aks/ccp/msi-adapter:master.230208.1"
+          imagePullPolicy: IfNotPresent
+          env:
+          - name: SUBSCRIPTION_ID
+            value: "{{ .Values.global.commonGlobals.Customer.SubscriptionId }}"
+          - name: RESOURCE_GROUP_NAME
+            value: "{{ .Values.global.commonGlobals.Customer.ResourceGroupName }}"
+          - name: MANAGED_CLUSTER_NAME
+            value: "{{ .Values.global.commonGlobals.Customer.ManagedClusterName }}"
+          - name: CONTROL_PLANE_ID
+            value: "{{ .Values.global.commonGlobals.Customer.Namespace }}"
+          - name: CLIENT_NAME
+            value: ama-metrics
+          securityContext:
+           capabilities:
+             add:
+               - NET_ADMIN
+      affinity:
+        nodeAffinity:
+          # affinity to schedule on to ephemeral os node if its available
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              preference:
+                matchExpressions:
+                  - key: kubernetes.azure.com/mode
+                    operator: In
+                    values:
+                      - system
+            - weight: 50
+              preference:
+                matchExpressions:
+                  - key: azuremonitor/metrics.replica.preferred
+                    operator: In
+                    values:
+                      - "true"
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: kubernetes.io/os
+                    operator: In
+                    values:
+                      - linux
+                  - key: type
+                    operator: NotIn
+                    values:
+                      - virtual-kubelet
+                  - key: kubernetes.azure.com/cluster
+                    operator: Exists
+      volumes:
+        - name: settings-vol-config
+          emptyDir: { }
+        - name: kubeconfig
+          secret:
+            secretName: kubeconfig-file
+        - name: prometheus-config-vol
+          configMap:
+            name: ama-metrics-prometheus-config
+            optional: true
+        - name: host-log-containers
```

Comment:
remove host-log-containers and host-log-pods mounts to align with ccp plugin chart?